front_end/panels/ai_chat/ui/settings/providers/BaseProviderSettings.ts

```
@@ -0,0 +1,78 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import type {
+  ModelOption,
```

ESLint

[error] 'ModelOption' is defined but never used.

(@typescript-eslint/no-unused-vars)